### ISSUE TITLE
NTR: VWF autoantibodies present (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8723]
New term request VWF autoantibodies present (elthat_100610010112): Von Willebrand factor

**Comment by pnrobinson** [comment]
@MickeySegal
I wonder if these are always autoantibodies or are against exogenous vWF? What is the context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK7014/ "VWF:Ab. All assays that are based on the binding of a monoclonal antibody (mAb) to a VWF A1 domain epitope"

The article goes on to discuss alloantibodies as a complication of treatment, but the discussion above is in the context of diagnosis, so appears to be autoantibodies.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000676
- label: Anti-von Willebrand factor antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:23297130
- def: The presence of autoantibodies in the serum that react to von Willebrand factor (VWF).
- comment: This feature is usually seen in the context of Von Willebrand disease (VWD), the most common inherited bleeding disorder. VWD is caused by deficiency or dysfunction of the multimeric glycoprotein, von Willebrand factor (VWF), and is clinically characterized by excessive mucocutaneous bleeding as well as musculoskeletal bleeding in type 3 VWD, the most severe form. Treatment options include infusion of concentrates of VWF, which in some patients leads to the production of alloantikbodies.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8723
- synonyms:
